soc/arm/nordic_nrf/nrf91/dts_fixup.h

```diff
@@ -32,10 +32,13 @@
 #define DT_SPI_2_NAME			DT_NORDIC_NRF_SPI_SPI_2_LABEL
 #define DT_SPI_3_NAME			DT_NORDIC_NRF_SPI_SPI_3_LABEL
 
-#define DT_WDT_0_NAME                   DT_NORDIC_NRF_WATCHDOG_WDT_0_LABEL
+#define DT_WDT_0_NAME			DT_NORDIC_NRF_WATCHDOG_WDT_0_LABEL
 
-#define DT_TIMER_0_NAME                 DT_NORDIC_NRF_TIMER_TIMER_0_LABEL
-#define DT_TIMER_1_NAME                 DT_NORDIC_NRF_TIMER_TIMER_1_LABEL
-#define DT_TIMER_2_NAME                 DT_NORDIC_NRF_TIMER_TIMER_2_LABEL
+#define DT_TIMER_0_NAME			DT_NORDIC_NRF_TIMER_TIMER_0_LABEL
```

Comment:
Looks like indentation is not correct here.

Reply:
nope, i've corrected it (previously those entries had spaces instead of tabs like the others).

Reply:
Yeap, weird github issue, i think.